src/Routes/RosDetailsPage/RosDetailsPage.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { useParams, withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { PageHeader } from '@redhat-cloud-services/frontend-components/PageHeader';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+//import { fetchRosDetails } from '../../Components/RosTable/redux/actions';
+import { routes as paths } from '../../../package.json';
+
+import { InventoryDetailHead, AppInfo, DetailWrapper } from '@redhat-cloud-services/frontend-components/Inventory';
+import { useSelector } from 'react-redux';
+
+const RosDetail = () => {
+    const { inventoryId } = useParams();
+    const displayName = useSelector(
+        ({ entityDetails }) => entityDetails?.entity?.display_name
+    );
+    // do we need a Reducer similar to groupsDetailReducer for entity ?
+
+    useEffect(() => {
+        insights.chrome?.hideGlobalFilter?.(true);
+        insights.chrome.appAction('system-detail');
+    }, []);
+
+    useEffect(() => {
+        insights?.chrome?.appObjectId?.(inventoryId);
+    }, [inventoryId]);
+
+    return (
+        <DetailWrapper
+            hideInvLink
+            showTags
+            //onLoad={({ mergeWithDetail }) => {
```

Comment:
This is required, otherwise the detail won't load. You can import `register` function from `src/store/index.js `.
```JS
register({ ...mergeWithDetail() })
```